### ISSUE TITLE
self.mapping is None even after adding --extra-files index_to_name.json 

### ISSUE THREAD

**Issue by festinais** [pytorch/serve#1728]
### 🐛 Describe the bug

I created the MAR file with this command:

`torch-model-archiver --model-name chest_limbs_spine --serialized-file model.pt --extra-files index_to_name.json --handler model_handler.py --requirements-file requir/requirements.txt --version 1.0`

I added the `--extra-files index_to_name.json` part where the json file contains some dummy json like: 
`{"0": "cat","1": "dog"}`

However, inside the method `postprocess` of my model_handler, I print `self.mapping` and it is always None.

From the documentation I'm supposed to see whatever is inside the file index_to_name.json, and this is not the case!
Any help appreciated!

This is my model handler: 

`

    def __init__(self):
        super().__init__()
        self._context = None
        self.initialized = False
        self.explain = False
        self.target = 0
        self.transforms = transforms.Compose([
            transforms.ToTensor(),
            NormalizeIntensity()
        ])

    def initialize(self, context):
        """
        Initialize model. This will be called during model loading time
        :param context: Initial context contains model server system properties.
        :return:
        """
        self._context = context
        self.initialized = True
        #  load the model, refer 'custom handler class' above for details
        properties = context.system_properties
        self.model = torch.jit.load("model.pt")

    def preprocess_one_image(self, req):
        """
        Process one image
        :param req:
        :return:
        """
        image = req.get("data")
        if image is None:
            image = req.get("body")
        print("image_data: ", image)
        image = Image.open(io.BytesIO(base64.decodebytes(bytes(image, "utf-8"))))
        image = self.transforms(image)

        # image = Image.open(io.BytesIO(image))
        # image = self.transforms(image)

        image = image.unsqueeze(0)
        return image

    def preprocess(self, requests):
        """
        Transform raw input into model input data.
        :param batch: list of raw requests, should match batch size
        :return: list of preprocessed model input data
        """
        # Take the input data and make it inference ready
        images = [self.preprocess_one_image(req) for req in requests]
        images = torch.cat(images)
        return images

    def inference(self, model_input):
        """
        Internal inference methods
        :param model_input: transformed model input data
        :return: list of inference output in NDArray
        """
        # Do some inference call to engine here and return output
        outs = self.model(model_input)
        probs = F.softmax(outs, dim=1)
        print("probabilities:: ", probs)
        preds = torch.argmax(probs, dim=1)
        return preds

    def postprocess(self, probs):
        """
        Return inference result.
        :param inference_output: list of inference output
        :return: list of predict results
        """
        # Take output from network and post-process to desired format
        # postprocess_output = inference_output
        res = []

        print("self mapping", self.mapping)

    def handle(self, data, context):
        """
        Invoke by TorchServe for prediction request.
        Do pre-processing of data, prediction using model and postprocessing of prediciton output
        :param data: Input data for prediction
        :param context: Initial context contains model server system properties.
        :return: prediction output
        """
        self.initialize(context)
        model_input = self.preprocess(data)
        model_output = self.inference(model_input)
        return self.postprocess(model_output)
`

### Error logs

no error logs

### Installation instructions

Installed torchserve with pip. 
Not using docker.

### Model Packaing

Put the info above.

### Versions

torchserve 0.6

### Repro instructions

Info above

**Comment by lxning** [comment]
@festinais could you first verify if index_to_name.json is included in mar file? For example, run "unzip mar_file" and then check if index_to_name.json is there.

Second, according to the code, I don't know which function reads index_to_name.json. Please check the model implementation if it is read by model

**Comment by maaquib** [comment]
@festinais Just to add to what @lxning already mentioned, `index_to_name.json` mapping is supported in the [default handler](https://github.com/pytorch/serve/blob/master/ts/torch_handler/base_handler.py#L108-L109). If you are using a custom handler, you'd have to add the mapping code in your handler along-with packaging it in the mar file
